src/services/BriefData.service.js

```diff
@@ -0,0 +1,14 @@
+import axios from "axios";
+
+const briefsUrl = `${process.env.REACT_APP_API_URL}/briefs`;
+const token = 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855760c11cd34f37e57d397dc2eeaefa68a';
```

Comment:
it is not considered a good practice to keep tokens hardcoded. They will now be kept on github in a commit history, so anyone in the future who has access to the repo can know your access token